### PR TITLE
DOC: add release instructions

### PR DESCRIPTION
**Pull request by untzag**
Attempting to add a how-to for releasing bluesky. Copied other content from `.github/CONTRIBUTING.md`.

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Describing the motivation for a change, the nature of a bug for bug fixes
+or some details on what an enhancement does are also good to include in a
+commit message. Messages should be understandable without looking at the code
+changes.
+
+Standard acronyms to start the commit message with are:
+
+* API: an (incompatible) API change
+* BLD: change related to building numpy
```
Comment (mrakitin) [inline docs/source/contributing.rst:56]:
```suggestion
* BLD: change related to building the project
```

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* Now push to your fork
+* Submit a `pull request <https://help.github.com/articles/using-pull-requests>`_
+
+At this point you're waiting on us. We like to at least comment on pull requests within three business days (and, typically, one business day).
+We may suggest some changes or improvements or alternatives.
+
+Hints to make the integration of your changes easy (and happen faster):
+* Keep your pull requests small
+* Don't forget your unit tests
+* All algorithms need documentation, don't forget the .rst file
```
Comment (mrakitin) [inline docs/source/contributing.rst:81]:
```suggestion
* All features need documentation, don't forget the .rst file
```

**Review by mrakitin — COMMENTED** [review]
Nice addition, @untzag! I added a couple of small suggestions.

**Review by mrakitin — APPROVED** [review]

**Comment by untzag** [comment]
I'm just now recognizing that I don't know exactly how `__version__` gets populated. Is that a step that needs to be documented here, or is that handled automatically by versioner?

**Comment by prjemian** [comment]
versioneer does it (when called from setup.py)

**Pull request merged**